### ISSUE TITLE
Interface field type error after upgrading from Gatsby v2 to v5 and using V 1.14

### ISSUE THREAD

**Issue by sstanfSD** [wp-graphql/wp-graphql#2760]
### Description

Ran into an interface field type error when running any builds after upgrading from Gatsby v2 to v5.
I was able to run develop, just not any builds.

**WP Gatsby Version**
2.3.3

![Screen Shot 2023-03-11 at 11 43 09 AM](https://user-images.githubusercontent.com/83233669/224463833-9afaf756-f5f2-44bb-8350-0ed6f3ba77cf.png)

### Steps to reproduce

`async function getPosts({ graphql, reporter }) {
  const graphqlResult = await graphql(`query WpPosts {
  allWpPost(sort: {date: DESC}) {
    edges {
      previous {
        id
      }
      post: node {
        id
        uri
      }
      next {
        id
      }
    }
  }
}
`

### WPGraphQL Version

1.14

### WordPress Version

6.1.1

### PHP Version

8.0

### Please confirm that you have searched existing issues in the repo.

- [X] Yes

### Please confirm that you have disabled ALL plugins except for WPGraphQL.

- [ ] Yes
- [ ] My issue is with compatibility with a specific WordPress plugin, and I have listed all my installed plugins (and version info) above.

**Comment by tomstorms** [comment]
Temporary solution I found was to downgrade WPGraphQL from v1.14 to v1.13.10 and worked for me

**Comment by sstanfSD** [comment]
Downgrading to v1.13.10 worked for me as well.

**Comment by sstanfSD** [comment]
> Temporary solution I found was to downgrade WPGraphQL from v1.14 to v1.13.10 and worked for me

This worked for me as well.

**Comment by witcradg** [comment]
Where can I get v1.13.10 ?

**Comment by justlevine** [comment]
> Where can I get v1.13.10 ? https://github.com/wp-graphql/wp-graphql/releases/tag/v1.13.10

@witcradg it's on the page you linked to, listed as `wp-graphql.zip`.

Direct link: https://github.com/wp-graphql/wp-graphql/releases/download/v1.13.10/wp-graphql.zip

**Comment by plavet** [comment]
Getting the same error, it kinda works on first build when you clear the cache, but if you change something in wordpress it triggers the same error again, then again quit develop process, clear cache manually, and it works fine on first build again. Sorry dont have it online, but in my gatsby-config i have DEV_SSR flag to true, and for the gatsby-source-wordpress plugin options i have
```js
options: {
        production: {
          allow404Images: true,
        },
        url: 'local-test-url/graphql',
        verbose: false,
        debug: {
          preview: true,
        },
        schema: {
          timeout: 60000,
          requestConcurrency: 50,
          previewRequestConcurrency: 50,
          perPage: 200,
        },
      },
```
Also in wordpress i have Enable GraphQL Debug Mode turned on

Using version v1.13.10 am getting this error
```bash
Error: ENOENT: no such file or directory, open '\node_modules\gatsby-source-wordpress\src\steps\create-schema-customization\transform-fields\transform-object.js'
```

followed by bunch of these errors
```bash
ERROR #85925  GRAPHQL.VALIDATION

There was an error in your GraphQL query:

Cannot return null for non-nullable field WpNodeWithFeaturedImageToMediaItemConnectionEdgeType.node.

Plugin: none

See our docs page for more info on this error: https://gatsby.dev/creating-type-definitions
```

it finishes build process with this warning
```bash
warn `isModuleDeclaration` has been deprecated, please migrate to `isImportOrExportDeclaration`
    at isModuleDeclaration (node_modules\@babel\types\lib\validators\generated\index.js:3940:35)
    at PluginPass.Program (node_modules\babel-plugin-lodash\lib\index.js:102:44)
```

PHP is 8.1.2, Node is 18.12.0, wordpress is 6.1.1, WP Gatsby is 2.3.3

I've reverted back to v1.13.8, wich was my previous version before updating to v1.14, all was runing fine, but now getting same errors and warning as for version v.1.13.10. Gonna try again tommorow to fix v1.13.8 to work again maybe pc restart will do the trick :)

Hope this helps

**Comment by TylerBarnes** [comment]
lets track this over in https://github.com/gatsbyjs/gatsby/issues/37729 since this is a `gatsby-source-wordpress` problem
